packages/proxyIdentity/contracts/IdentityManager.sol

```diff
@@ -0,0 +1,45 @@
+pragma solidity ^0.5.0;
+
+
+contract IdentityManager {
```

Comment:
If this contract is just doing the events, maybe it's more specific to call it `IdentityEvents` or maybe `IdentityNotifier`

Reply:
I plan to expand this contract in the future, such as adding methods to return a list of assets or may be batch asset creation